packages/lisk-p2p/src/p2p.ts

```diff
@@ -126,15 +132,21 @@ export class P2P extends EventEmitter {
 			if (!this._triedPeers.has(peerId)) {
 				this._triedPeers.set(peerId, peerInfo);
 			}
-			this.emit(EVENT_CONNECT_OUTBOUND);
+			this.emit(EVENT_CONNECT_OUTBOUND, peerInfo);
```

Comment:
If this will be emitted, then the user will know which peer we are making an outbound connection with. Do you think its okay or we should not emit this info?

Reply:
We can expose it for doing analysis and integration tests but I agree that it's not a very important event. We could leave it out of the main documentation.